scripts/exrrfs_jedivar.sh

```diff
@@ -147,7 +147,7 @@ export err=$?
 err_chk
 #
 # ncks increments to cold_start IC
-if [[ ${start_type} == "cold" ]]; then
+if [[ ${start_type} == "cold" && ${COLDSTART_CYCS_DO_DA} == "true" ]]; then
   var_list="pressure_p,rho,qv,qc,qr,qi,qs,qg,ni,nr,ng,nc,nifa,nwfa,volg,surface_pressure,theta,u,uReconstructZonal,uReconstructMeridional"
   ncks -A -H -v "${var_list}" ana.nc mpasin.nc
   export err=$?
```

Comment:
@ShunLiu-NOAA, this was the part I was referring to this morning. At cold start cycles the analysis file is called `ana.nc` in the jedivar.yaml and a other times the analysis file is `mpasin.nc`. I'm not sure on the exact reason for this, but with the extra check for `COLDSTART_CYCS_DO_DA == "true"` ensures that any changes to `ana.nc` do not get merged into the `mpasin.nc` file. 

In summary, at 00/12Z cold start times, airTemperature, specificHumidity, and winds from aircar (133/233) get set to passivate (aka monitor), JEDI is still run but nothing is impacting the analysis (`ana.nc`), and to make double sure, the result from `ana.nc` does not get applied to `mpasin.nc`. Therefore, `mpasin.nc` will be exactly as if JEDI had truly been skipped.

Reply:
Just to follow up. I `nccmp -dm` (compares data and metadata) and `mpasin.nc` was identical before and after running JEDI.